File-Formats/DocIO/word-to-image.md

```diff
@@ -137,6 +138,203 @@ using (Stream docStream = typeof(App).GetTypeInfo().Assembly.GetManifestResource
+    'Set the scaling mode for charts (Normal mode reduces the file size).
+    wordDocument.ChartToImageConverter.ScalingMode = ScalingMode.Normal
+    'Convert the entire Word document to images.
+    Dim images As Image() = wordDocument.RenderAsImages(1, 2, ImageType.Bitmap)
+    Dim i = 0
+    For Each image As Image In images
+        'Save the image as jpeg.
+        image.Save("WordToImage_" & i & ".jpeg", ImageFormat.Jpeg)
+        i += 1
+    Next
+End Using
+{% endhighlight %}
+
+{% highlight c# tabtitle="UWP" %}
+//DocIO supports Word to image conversion in Windows Forms, WPF, ASP.NET and ASP.NET MVC platform alone
+{% endhighlight %}
+
+{% highlight c# tabtitle="ASP.NET Core" %}
+//Open the file as Stream.
+using (FileStream docStream = new FileStream("Template.docx", FileMode.Open, FileAccess.Read))
+{
+    //Load file stream into Word document.
+    using (WordDocument wordDocument = new WordDocument(docStream, FormatType.Docx))
+    {
+        //Create a new instance of DocIORenderer class.
+        using (DocIORenderer render = new DocIORenderer())
+        {
+            //Convert a specific range of pages in Word document to images.
+            Stream[] imageStreams = wordDocument.RenderAsImages(1, 2); 
+            int i = 0;
+            foreach (Stream stream in imageStreams)
+            {
+                //Reset the stream position.
+                stream.Position = 0;
+                //Save the stream as file.
+                using (FileStream fileStreamOutput = File.Create("WordToImage_" + i + ".jpeg"))
+                {
+                    stream.CopyTo(fileStreamOutput);
+                }
+                i++;
+            }
+        }
+    }
+}
+{% endhighlight %}
+
+{% highlight c# tabtitle="Xamarin" %}
+//Open the file as Stream.
+using (Stream docStream = typeof(App).GetTypeInfo().Assembly.GetManifestResourceStream("Sample.Assets.Template.docx"))
+{
+    //Load file stream into Word document.
+    using (WordDocument wordDocument = new WordDocument(docStream, FormatType.Docx))
+    {
+        //Create a new instance of DocIORenderer class.
+        using (DocIORenderer render = new DocIORenderer())
+        {
+            //Convert the entire Word document to images.
+            Stream[] imageStreams = wordDocument.RenderAsImages(1, 2);
+            int i = 0;
+            foreach (Stream stream in imageStreams)
+            {
+                //Reset the stream position.
+                stream.Position = 0;
+                //Save the stream as file in the device and invoke it for viewing.
+                Xamarin.Forms.DependencyService.Get<ISave>().SaveAndView("WordToImage_" + i + ".jpeg", "image/jpeg", stream as MemoryStream);
+                i++;
+            }
+        }
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Word-to-Image-conversion/Specific-range-of-pages-Word-to-image).
```

Comment:
Add the below sentence in existing note

DocIO supports Word to image conversion in UWP application using DocIORenderer.

Reply:
Addressed